src/components/solverComponents/SAT_Components/SAT_Help.tsx

```diff
@@ -0,0 +1,7 @@
+import { Text } from "@chakra-ui/react";
+
+export const SAT_Help = () => {
+    return (
+        <Text>there is no help where you are going</Text>
```

Comment:
an actually helpful text would be nice :)

Reply:
This exercise is left to the reader.


(Will be added when more functionality is implemented)